Hakawai/Mentions/HKWMentionsPluginV2.m

```diff
@@ -535,10 +526,17 @@ - (void)bleachExistingMentionAtRange:(NSRange)range {
         for (NSString *key in unselectedAttributes) {
             [buffer removeAttribute:key range:HKW_FULL_RANGE(input)];
         }
+        // Restore parent font/color to text
+        UIFont *parentFont = parentTextView.fontSetByApp;
+        UIColor *parentColor = parentTextView.textColorSetByApp;
+        if (parentFont) {
+            [buffer addAttribute:NSFontAttributeName value:parentFont range:HKW_FULL_RANGE(input)];
+        }
+        if (parentColor) {
+            [buffer addAttribute:NSForegroundColorAttributeName value:parentColor range:HKW_FULL_RANGE(input)];
+        }
```

Comment:
can we rename `typingAttributesByStrippingMentionAttributes` to `nonMentionTextAttribute` and then reuse that attribute for the buffer? so we can make sure it's consistent